app/scripts/lib/cropper.js

```diff
@@ -71,22 +74,28 @@ function (_) {
     this.slider.on('input', function () {
       self.resize(parseInt(this.value, 10));
     });
+    this.slider.on('change', function () {
+      self.metrics.logEvent('avatar-cropper.zoom.range');
```

Comment:
@zaach - IIRC, you wanted to keep this particular module pretty "pure" so that it could be extracted into its own library. Adding metrics will destroy that purity since metrics are very specific to our app.

If we are going to integrate this more fully with the app, I think we should turn it into a Backbone based view and use all the functionality available to us to reduce boilerplate, if not, then let's try to move the metrics out of here somehow.